### PR TITLE
data: SebastianJames55

### PR DESCRIPTION
**Pull request by SebastianJames55**

### CODE REVIEW

**Review by shyamtawli — APPROVED** [review]
LGTM

File: data/SebastianJames55.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "links": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/call_me_tien",
+            "icon": "FaTwitter"
+        },
+        {
+            "name": "LinkedIn",
+            "url": "https://www.linkedin.com/in/sebastianjames55/",
+            "icon": "FaLinkedIn"
```
Comment (Sindhuinti) [inline data/SebastianJames55.json:23]: Please change icon name to `FaLinkedin`
Docs : https://linkfree.io/icons

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/SebastianJames55.json (hunk 1 above)
Comment (SandeepKrSuman) [inline data/SebastianJames55.json:23]:
```diff
- "icon": "FaLinkedIn"
+ "icon": "FaLinkedin"
```

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]
Hey @SebastianJames55 , thanks for your contribution. 
Please make the suggested change. Also, you may add more socials (it's up to you). Everything else looks good to me.

File: data/SebastianJames55.json (hunk 1 above)
Comment (SebastianJames55) [reply data/SebastianJames55.json:23]: Thanks for pointing this out, I missed out this. I'll correct & commit.

**Review by Sindhuinti — APPROVED** [review]

**Review by SandeepKrSuman — APPROVED** [review]
Thanks, @SebastianJames55 
Looks good! 🚀

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge.

**Pull request merged**